### DISCUSSION TITLE
Zigbee Map and device connection to the Mesh network problems

### DISCUSSION THREAD

**Discussion by nmaou** [arendst/Tasmota#17726] [General]
Hello, I have 2 questions regarding the zigbee mesh network.
I am using the sonoff Zigbee bridge. The first version of the bridge.
All connected devices are SonOff zigbee devices.
1. Can I manually connect devices to each other or to a router;
2. In the zigbee map not all devices are shown connected as you can see in the picture below. The devices in the red frames are part of the network but they are not shown connected to a router. Why is that; How can I find out to which device these devices are connected to;
Any help is welcomed.
Napoleon Maou
![2023-01-17_10-21-23](https://user-images.githubusercontent.com/8890082/212849730-3f5a788b-0da2-4352-9010-49bf0559fe02.png)

**Comment by s-hadinger** [comment]
This only means that no router claims to have those devices as children. You can't choose which router a device uses and the Zigbee network periodically and automatically reconfigures it's topology. 

You can try to request the device directly with `Zbmap <device>` right after waking it up. This will also update the map for this device.
